cli/ceph/orch/orch.py

```diff
@@ -0,0 +1,82 @@
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class OrchApplyCommandError(Exception):
+    pass
+
+
+class Orch(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(Orch, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} orch"
+
+    def ls(self, **kw):
+        """
+        Lists the status of the given services running in the Ceph cluster.
+
+        Args:
+            kw (Dict): Key/value pairs that needs to be provided to the installer.
+            Example:
+                Supported keys:
+                    service_type(str): type of service(mon, osd, mgr, mds, rgw)(Optional).
+                    service_name(str): name of host(Optional).
+                    export(bool): Whether to export the service specifications knows to
+                                  the orchestrator(Optional).
+                    format(str): the type to be formatted(yaml)(Optional).
+                    refresh(bool): Whether to refresh or not(Optional).
+        """
+        cmd = f"{self.base_cmd} ls {build_cmd_from_args(**kw)}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def apply(self, service_name, args=None, **kw):
+        """
+        Applies the configuration to hosts.
+        Args:
+          service_name : Name of the service
+          args (list): Supporting args for service apply
+          kw (Dict): Key/value pairs that needs to be provided to the installer.
+          Example:
+            Supported keys:
+              hosts(list): name of host(Optional).
+              placement(dict): Placement on hosts(Optional).
+              count(int): No of hosts to be applied(Optional)
+        Returns:
+          Dict(str):  A mapping of host strings of the given task's return
+                      value for that host's execution run.
+        """
+        cmd = f"{self.base_cmd} apply {service_name}"
+        for arg in args:
+            cmd += f" {arg}"
+        cmd += build_cmd_from_args(**kw)
+        log.info(f"Running command {cmd}")
+        out, _ = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def rm(self, service_name):
+        """
+        Removes a service.
+        Args:
+          service_name (Str): Name of the service to be removed, if not specified
```

Comment:
Description doesn't match with the logic. This is mandatory parameter then why conditional comment here ?